File: c/2019/09/27/1.py
print("程序中有存储一个数字，你可以尝试一下猜出这个数字")
while True:
    a=int(input("请输入您猜的数字："))
    if a==12:
        print("您猜对了")
        break
    else:
        if a<12:
            print("您猜的数字小了")
        else:
            print("您猜的数字大了")

File: c/2019/10/17/1.cpp
#include<stdio.h>
int main(){
    int sum,term,n,a;
    printf("Please input a:\n");
    scanf("%d",&a);
    printf("Pleas intput n:\n");
    scanf("%d",&n);
    term=a;
    for(int i=1;i<=n;i++){
        if (i==1){
            printf("%d",term);
            sum=term;
        }
        else
        {
            term=term*10+a;
            sum=sum+term;
            printf("+%d",term);
        }
    }
    printf("=%d",sum);
}